Packs/Respond/IncidentFields/incidentfield-Asset_Criticality.json

```
@@ -1,18 +1,29 @@
 {
- "associatedTypes": [
-  "Respond Software Incident"
- ],
- "caseInsensitive": true,
- "cliName": "assetcriticality",
- "editForm": true,
- "id": "incident_assetcriticality",
- "name": "Asset Criticality",
- "threshold": 72,
- "type": "shortText",
- "version": -1,
- "fromVersion": "5.0.0",
- "content": true,
- "system": false,
- "required": false,
- "group": 0
+    "associatedToAll": false,
+    "associatedTypes": [
+        "Respond Software Incident"
+    ],
+    "caseInsensitive": true,
+    "cliName": "assetcriticality",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_assetcriticality",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "Asset Criticality",
+    "neverSetAsRequired": false,
+    "ownerOnly": false,
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "shortText",
+    "unmapped": false,
+    "unsearchable": false,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "5.0.0"
```

```suggestion
    "fromVersion": "6.0.0"
```

Please fix it in all relevant places.